Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Person.hs

```diff
@@ -629,7 +629,7 @@ getNearestDrivers mbVariant LatLong {..} radiusMeters merchantId onlyNotOnRide m
               sedanResult = getResult SEDAN $ variant == SEDAN || (variant == SUV && canDowngradeToSedan)
```

Comment:
Needs to handle the same in these functions `buildFullDriverListOnRide`, `buildFullDriverList` and `getNearestDriversCurrentlyOnRide` too